build.ps1

```diff
@@ -41,14 +42,14 @@ http://cakebuild.net
 [CmdletBinding()]
 Param(
     [string]$Script = "build.cake",
-    [string]$Target = "Default",
-    [ValidateSet("Release", "Debug")]
-    [string]$Configuration = "Release",
+    [string]$Target,
+    [string]$Configuration,
     [ValidateSet("Quiet", "Minimal", "Normal", "Verbose", "Diagnostic")]
-    [string]$Verbosity = "Verbose",
+    [string]$Verbosity,
+    [switch]$ShowDescription,
+    [Alias("WhatIf", "NoOp")]
```

Comment:
Sorry, missed this! To play devil's advocate, hyphenated words do capitalize where the hyphen is removed. But I don't think this was intentional.